scripts/release/uniprot_update/uniprot_xml2sql_isoform.pl

```diff
@@ -1010,6 +1013,31 @@ sub update_chain_log {
     }
 }
 
+sub is_sequence_changed_attribute_needs_updating {

```

Comment:
Maybe provide some comments for this function?

Reply:
Yeah, after reviewing this function I realized it is probably trying to do too many things.  I'm going to refactor a bit to make this logic clearer and I will add comments to the end product.